lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
Setting the `remove_bomb()` at the top of the method does that, too, though I don't think we need to go to great lengths to prevent it. One thing I think would be nice is for the user to also be able to add their own bomb types, which is more challenging if we're generating the list on the fly every time. As implemented here, it's easy for a user to say `StinkBomb.bombs << MyCustomBomb.new`.